src/components/tool_tip/tool_tip.tsx

```diff
@@ -47,6 +47,13 @@ const positionsToClassNameMap: { [key in ToolTipPositions]: string } = {
 
 export const POSITIONS = keysOf(positionsToClassNameMap);
 
+const displayToClassNameMap = {
+  inlineBlock: undefined,
+  block: 'euiToolTip--displayBlock',
```

Comment:
Because this will then be applied to the anchor, the class name needs to correlate with that.

```suggestion
  block: 'euiToolTipAnchor--displayBlock',
```